app/MMB/src/scenes/list/cityImage/StatusBadge.js

```diff
@@ -66,13 +104,36 @@ export default createFragmentContainer(
 
 const styles = StyleSheet.create({
   confirmBadge: {
-    backgroundColor: 'rgba(51, 189, 27, 0.6)',
+    backgroundColor: Color.green.normal,
+    opacity: 0.7,
```

Comment:
We need to reuse style. If they all use `opacity: 0.7`, then we can have:

```js
style={[styles.badge, styles.someColor]}
```

Where:

```js
badge {
  opacity: 0.7,
}
```

Also you have a lot of styles that are exactly the same. Instead of per badge, you can have, `styles.black`, `styled.red` and so on... Then, from the other commend I made:

```js
const statuses = {
   // Map object
   CANCELLED: {
     translation: 'your-value-here'
     style: [styles.badge, styles.red]
   }
  // Other status
  ...
}
```